start-docker-compose.sh

```diff
@@ -87,6 +86,7 @@ fi
 if [ "$buildZac" = "true" ]; then
     echo "Building ZAC Docker Image ..."
     ./gradlew buildDockerImage
+    export ZAC_DOCKER_IMAGE=ghcr.io/infonl/zaakafhandelcomponent:dev
```

Comment:
I would prefer to see the image hash - that will make it super explicit, but that will then have to be "read" from somewhere. But don't bother with that right now 😄